index.html

```diff
@@ -13,10 +13,15 @@
       <div class="logo">
          <a href="index.html">My Logo</a>
       </div>
-      <nav class="hamburger">
-         <span></span>
-         <span></span>
-         <span></span>
+      <nav class="nav">
```

Comment:
The navbar is disappearing under the 768px. Kindly fix that to have a responsive web page. Check the image below. 🦾

<img width="749" alt="Screenshot 2022-12-16 at 12 19 05 AM" src="https://user-images.githubusercontent.com/25887253/207969708-b5c5ad87-aa23-4d37-9639-4b7bd81de279.png">

Reply:
Thanks so much for the feedback. I will ensure I go through all your comments and make adjustments where necessary.